uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerCompanyPortal.java

```diff
@@ -263,7 +239,7 @@ public void enrollDevice(@NonNull final String username,
                 "com.microsoft.windowsintune.companyportal:id/setup_title"
         );
 
-        if (!setupCompletePage.exists()) {
+        if (!setupCompletePage.waitForExists(ENROLLMENT_PAGE_TIMEOUT)) {
```

Comment:
Why does this need a different timeout from the rest of the code that seems to use FIND_UI_ELEMENT_TIMEOUT_LONG?

Reply:
We're actually using this timeout more often than the longer one. The longer one is a bit overkill for this case iirc